### PR TITLE
Bulk then execute fixes

### PR DESCRIPTION
**Pull request by brycelelbach**
These are some editorial fixes for `bulk_then_execute`'s wording.

### CODE REVIEW

File: wording.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ In the Table below, `x` denotes a (possibly const) executor object of type `X` a
...
+| `x.execute(f)` | `void` | Creates an execution agent which invokes `DECAY_COPY(std::forward<F>(f))()` at most once, with the call to `DECAY_COPY` being evaluated in the thread that called `execute`. <br/> <br/> May block pending completion of `DECAY_COPY(std::forward<F>(f))()`. <br/> <br/> The invocation of `execute` synchronizes with (C++Std [intro.multithread]) the invocation of `f`. <br/> <br/> `execute` shall not propagate any exception thrown by `DECAY_COPY(std::forward<F>(f))()` or any other function submitted to the executor. [*Note:* The treatment of exceptions thrown by one-way submitted functions and the forward progress guarantee of execution agents created by one-way execution functions are specific to the concrete executor type. *--end note.*] |
```
Comment (jaredhoberock) [inline wording.md]: Is the only change to this (and the following paragraphs) to eliminate extraneous whitespace in the `DECAY_COPY` call?

Comment (brycelelbach) [reply wording.md]: Yes (sorry, I thought that GitHub showed line-to-line diffs). It's just replacing "DECAY_COPY( " with "DECAY_COPY(".

**Comment by jaredhoberock** [comment]
Thanks, so it looks like these are all editorial fixes (they all look OK to me), right? If so, I think we can just go ahead and merge in order to avoid spending time during a call.

**Comment by brycelelbach** [comment]
That sounds fine to me. All of these changes are purely editorial.

Do you want to rebase it, or should I?

**Comment by jaredhoberock** [comment]
Thanks, I rebased to R8 and then merged.

**Comment by brycelelbach** [comment]
Needs to go into master too, right?

**Pull request merged**